xarray/plot/utils.py

```diff
@@ -354,3 +355,28 @@ def get_axis(figsize, size, aspect, ax):
         ax = plt.gca()
 
     return ax
+
+
+def label_from_attrs(da):
+    ''' Makes informative labels if variable metadata (attrs) follows
+        CF conventions. '''
```

Comment:
Should this be a private method?

Reply:
I thought it might be useful to have it accessible.